app/src/main/java/com/io/movio/ui/movielist/adapter/MoviesAdapter.kt

```diff
@@ -3,24 +3,32 @@ package com.io.movio.ui.movielist.adapter
 import android.view.LayoutInflater
 import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.io.movio.data.enums.Genre
 import com.io.movio.databinding.MovieRowBinding
 import com.io.movio.data.models.Movie
 
-class MoviesAdapter(private val listener: ItemOnClickListener)
-    : RecyclerView.Adapter<MoviesAdapter.MovieViewHolder>() {
+private const val BASE_IMAGE_URL = "https://image.tmdb.org/t/p/w500"
+
+class MoviesAdapter(private val listener: ItemOnClickListener) :
+    RecyclerView.Adapter<MoviesAdapter.MovieViewHolder>() {
 
     private val movies = mutableListOf<Movie>()
 
-    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MovieViewHolder
-    = MovieViewHolder(MovieRowBinding.inflate(LayoutInflater.from(parent.context), parent, false))
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MovieViewHolder =
+        MovieViewHolder(MovieRowBinding.inflate(LayoutInflater.from(parent.context), parent, false))
 
     override fun onBindViewHolder(holder: MovieViewHolder, position: Int) {
 
         holder.apply {
             binding.tvTitle.text = movies[position].title
-            binding.tvGenre.text = movies[position].genre
+            binding.tvGenre.text = getGenreToString(movies[position].genre!!)
```

Comment:
It is not a good practise to use the !! too much as it can lead to crashes. Think of a way to avoid that